rusk/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Deprecate `[chain].min_deploy_points` config [#3341]
 - Deprecate `[chain].block_gas_limit` config [#3341]
 - Change how Rusk controls the archive for synchronization [#3359]
+- Change the gas limit for queries [#3481]
```

Comment:
This should be in the Fixes section, something like "fix unresponsive node when query bad contracts"